doc/releases/changelog-dev.md

```diff
@@ -337,6 +333,17 @@
 
 <h3>Breaking changes</h3>
 
+* Old circuit drawing infrastructure is being deprecated. This includes the following methods and classes:
```

Comment:
This entry may fit the `Deprecations` section better, especially as these are all developer-facing functions/methods.